Cargo.lock

```diff
@@ -13365,18 +13401,18 @@ dependencies = [
 
 [[package]]
 name = "sqllogictest"
-version = "0.20.4"
```

Comment:
This looks suspicious. I don't see any `Cargo.toml` changes in this PR. Do we mistakenly include unnecessary changes in `Cargo.lock`?